lib/checks/aria/no-implicit-explicit-label.json

```diff
@@ -0,0 +1,11 @@
+{
+	"id": "no-implicit-explicit-label",
+	"evaluate": "no-implicit-explicit-label.js",
+	"metadata": {
+		"impact": "moderate",
+		"messages": {
+			"fail": "Fix at least one of the following: `aria-labelledby`, `aria-label`, `non-empty title`",
+			"incomplete": "Check that the <label> does not need be part of the ARIA {{=it.data}} field's name"
```

Comment:
Remove `fail`, and add a "pass" instead. Something like "There is no mismatch between a <label> and accessible name"